elasticsearch-core/src/test/scala/com/sumologic/elasticsearch/restlastic/RestlasticSearchClientTest.scala

```diff
@@ -928,6 +928,16 @@ class RestlasticSearchClientTest extends WordSpec with Matchers with ScalaFuture
       resFut.futureValue.rawSearchResponse.highlightAsMaps.head should be(Map("text" -> List("here")))
     }
 
+    "support check where a document exists by document id" in {
+      val doc = Document("doc0001", Map("text" -> "here"))
+      val indexFuture = restClient.bulkIndex(index, tpe, Seq(doc))
+      whenReady(indexFuture) { _ => refresh() }
+      // verify doc0001 exists
+      restClient.documentExistsById(index, tpe, "doc0001").futureValue should be(true)
+      // verify doc0002 does not exist
```

Comment:
probably don't need these two comments any more, now that we're returning `false` instead of the 404 stuff?